compiler/noirc_evaluator/src/ssa/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -584,7 +590,8 @@ impl<F: AcirField> AcirContext<F> {
     pub(crate) fn mul_var(&mut self, lhs: AcirVar, rhs: AcirVar) -> Result<AcirVar, RuntimeError> {
         let lhs_data = self.vars[&lhs].clone();
         let rhs_data = self.vars[&rhs].clone();
-        let result = match (lhs_data, rhs_data) {
+
+        let result = match (lhs_data.clone(), rhs_data.clone()) {
```

Comment:
```suggestion
        let result = match (lhs_data, rhs_data) {
```

Looks like this is left over from some reverted changes.

Reply:
I had local changes where these were removed so I updated manually